PocketKit/Sources/Sync/Source.swift

```diff
@@ -77,6 +77,10 @@ public protocol Source {
     func save(url: URL)
 
     func fetchItem(_ url: URL) -> Item?
+
+    func searchPremiumSaves(search: String) -> [SavedItem]?
+
+    func searchSaves(search: String) -> [SavedItem]?
```

Comment:
what do you think about consolidating this to one search function and use a boolean or `user.status` to determine what should be fetched?

Reply:
ok